File: verify/yosupo/sample/many_aplusb/io.test.py
# verification-helper: PROBLEM https://judge.yosupo.jp/problem/many_aplusb

import sys

def main():
    for _ in range(int(sys.stdin.readline())):
        a, b = map(int, sys.stdin.readline().split())
        print(a + b)


if __name__ == "__main__":
    main()

File: verify/yosupo/sample/many_aplusb/io.test.cpp
#define PROBLEM "https://judge.yosupo.jp/problem/many_aplusb"

#include <iostream>

int main() {
    std::ios_base::sync_with_stdio(false);
    std::cin.tie(0);
    int T;
    std::cin >> T;
    for (int i = 0; i < T; i++) {
        long long A, B;
        std::cin >> A >> B;
        std::cout << A + B << '\n';
    }
}
